src/main/java/org/apache/sysds/runtime/controlprogram/paramserv/LocalParamServer.java

```diff
@@ -33,24 +33,32 @@ public LocalParamServer() {
 
 	public static LocalParamServer create(ListObject model, String aggFunc, Statement.PSUpdateType updateType,
 		Statement.PSFrequency freq, ExecutionContext ec, int workerNum, String valFunc, int numBatchesPerEpoch,
-		MatrixObject valFeatures, MatrixObject valLabels)
+		MatrixObject valFeatures, MatrixObject valLabels,boolean modelAvg)
```

Comment:
all the additional parameters in these constructors are missing a space before the new parameter

Reply:
Thanks for the comment. the formatting is fixed.

Reply:
@atefeh-asayesh you don't need to comment on every one of these comments - just add the commits with the fixed and let me know once you're ready and I merge it in.